test/IntegrationTestCommon/Factories/WebApplicationFactoryBase.cs

```diff
@@ -72,6 +73,13 @@ protected override void ConfigureWebHost(IWebHostBuilder builder)
                 .AddJsonFile("appsettings.Development.json");
 
             c.AddUserSecrets(typeof(Identity.Startup).Assembly, optional: true);
+
+
+            var featureFlagDict = new Dictionary<string, object>
+            {
+                { FeatureFlagKeys.EmailVerification, true },
+            };
```

Comment:
is this used?

Reply:
Nope, it was a first attempt that I failed to clean up! Thanks for catching that!